open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/enterprise/repositoryconnector/EnterpriseOMRSMetadataCollection.java

```diff
@@ -60,18 +59,17 @@
      * Constructor ensures the metadata collection is linked to its connector and knows its metadata collection Id.
      *
      * @param enterpriseParentConnector connector that this metadata collection supports.  The connector has the information
-     *                        to call the metadata repository.
-     * @param repositoryName name of the repository used for logging.
-     * @param repositoryHelper class used to build type definitions and instances.
-     * @param repositoryValidator class used to validate type definitions and instances.
-     * @param metadataCollectionId unique Identifier of the metadata collection Id.
+     *                                  to call the metadata repository.
+     * @param repositoryName            name of the repository used for logging.
+     * @param repositoryHelper          class used to build type definitions and instances.
+     * @param repositoryValidator       class used to validate type definitions and instances.
+     * @param metadataCollectionId      unique Identifier of the metadata collection Id.
      */
     public EnterpriseOMRSMetadataCollection(EnterpriseOMRSRepositoryConnector enterpriseParentConnector,
-                                            String                            repositoryName,
-                                            OMRSRepositoryHelper              repositoryHelper,
-                                            OMRSRepositoryValidator           repositoryValidator,
-                                            String                            metadataCollectionId)
-    {
+                                            String repositoryName,
+                                            OMRSRepositoryHelper repositoryHelper,
```

Comment:
This change seems to have impacted the formatting throughout this file - there are many lines that have changed that are not related to the original update